### ISSUE TITLE
ERRORS in output

### ISSUE THREAD

**Issue by J-PC** [ansible-community/ansible-bender#313]
It seem's the problem have been reintroduced or some other output leads to the same consequence. At least the build and clean commands are affected.

![image](https://github.com/user-attachments/assets/d3fc7d3a-d5b5-486c-8512-e71bb0be58af)

_Originally posted by @J-PC in https://github.com/ansible-community/ansible-bender/issues/193#issuecomment-2346557686_

**Comment by TomasTomecek** [comment]
I don't think we ever fixed this, based on this comment https://github.com/ansible-community/ansible-bender/issues/193#issuecomment-555528054 and the fact there is no linked pull request.

**Comment by rienafairefr** [comment]
I think #195  is the PR that was supposed to fix this. But it's only for the "pull" command, and not the others like "from"
